Dockerfile.QA

```diff
@@ -46,7 +47,7 @@ FROM ${BUILD_IMAGE} AS build
 # Ensure apt-get won't prompt for selecting options
 ENV DEBIAN_FRONTEND=noninteractive
 
-WORKDIR /tmp/tritonbuild/tritonserver/build
+WORKDIR ${TRITONBUILD_DIR}/tritonserver/build
```

Comment:
You don't need to call `ARG TRITONBUILD_DIR` first to re-declare it?

Reply:
Good catch